src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -556,6 +556,11 @@ def _get_final_device_map(device_map, pipeline_class, passed_class_obj, init_dic
                 loaded_sub_model = passed_class_obj[name]
 
         else:
+            sub_model_dtype = (
+                torch_dtype.get(name, torch_dtype.get("default", torch.float32))
```

Comment:
Can't we just do
```
torch_dtype.get(name, torch.float32)
```

Reply:
Ah I see. Hmm user provided default makes sense. I think it would get a bit tedious to have to specify it for each component.